src/home_robot_sim/home_robot_sim/env/habitat_ovmm_env/habitat_ovmm_env.py

```diff
@@ -27,8 +29,24 @@
 from home_robot_sim.env.habitat_objectnav_env.visualizer import Visualizer
 
 
+class JointActionIndex(IntEnum):
+    """
+    Enum representing the indices of different joints in the action space.
+    """
+
+    # TODO: This needs to be common between sim and real as they share the same API for actions
+    ARM = 0  # A single value is used to control the extension
```

Comment:
If this needs to be constant, let's move it to motion/stretch?

Reply:
We can call this SimulationJointActionIndex

Reply:
Renamed